### DISCUSSION TITLE
Welcome to gnome-pomodoro Discussions!

### DISCUSSION THREAD

**Discussion by codito** [focustimerhq/FocusTimer#538] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Uglevod** [comment]
Hi, its very good app + was write on vala = its good fo my not very speed pc )  (- its not electron or other..)
and i see other pomodoro apps - like - Pomatez 
and it has - list of task\sub task 
and its wery good things for productivity - for me.
Can u add tasks and maybe some  statistic  -   how many minets i was in to what task 
in main app or on plugin 
Thx for  u dev  )

**Reply by josehbez** [reply]
I think it's great idea integrate with https://github.com/alainm23/planner
